ts/dynamodb.ts

```diff
@@ -9,6 +9,18 @@ const params: DynamoDB.DocumentClient.GetItemInput = {
     }
 };
 
```

Comment:
Can you add tests to make sure we can explicitly set a variable to the `DynamoDB.Converter` type? Also, would be nice to verify we can access `DocumentClient.ConverterOptions`.

Can be simple, like:
```javascript
const converter: Converter = DynamoDB.Converter;
// and a test for input with converter options
const converterOptions: DynamoDB.DocumentClient.ConverterOptions = {convertEmptyValues: true};
DynamoDB.Converter.input('string', converterOptions);
```

How hard would it be to also expose ConverterOptions on the Converter namespace? Just feels a little odd having to access it off the DocumentClient namespace instead.